DataFormats/L1TParticleFlow/interface/PFCandidate.h

```diff
@@ -0,0 +1,61 @@
+#ifndef DataFormats_L1TParticleFlow_PFCandidate_h
+#define DataFormats_L1TParticleFlow_PFCandidate_h
+
+#include <vector>
+#include "DataFormats/L1Trigger/interface/L1Candidate.h"
+#include "DataFormats/L1Trigger/interface/Muon.h"
+#include "DataFormats/L1TParticleFlow/interface/PFCluster.h"
+#include "DataFormats/L1TParticleFlow/interface/PFTrack.h"
+
+namespace l1t {
+
+  class PFCandidate : public L1Candidate {
+  public:
+    typedef edm::Ptr<l1t::Muon> MuonRef;
+    enum Kind { ChargedHadron = 0, Electron = 1, NeutralHadron = 2, Photon = 3, Muon = 4 };
```

Comment:
To fit with offline PF, better to name this `ParticleType`
(also the order is not the same as offline PF, but the order is also not consistent elsewhere in CMSSW)